matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/database/helper/ChunkEntityHelper.kt

```diff
@@ -86,10 +86,9 @@ internal fun ChunkEntity.add(roomId: String,
                              isUnlinked: Boolean = false) {
 
     assertIsManaged()
-    if (event.eventId.isNullOrEmpty() || this.events.fastContains(event.eventId)) {
+    if (event.eventId != null && events.fastContains(event.eventId)) {
```

Comment:
It's not doing the same thing (it's a fix?)

Reply:
Yep, it's a fix for handling empty event id (it happens inside invitation room timeline)